### PR TITLE
Redirect to new site -> click download cert in home page

### PR DESCRIPTION
**Pull request by glanttz**

### CODE REVIEW

File: src/components/CertsComponent.vue [hunk 1]
```diff
@@ -19,21 +24,28 @@
             type: String,
             default: '',
         },
-        typeOfDoc: {
```
Comment (PixelSculptor) [inline src/components/CertsComponent.vue:22]: is it path or link to place in db?

File: src/components/CertsComponent.vue [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
+            Pobierz
+        </button-component>
     </li>
 </template>
 
 <script setup>
     import { computed } from 'vue';
     import ButtonComponent from './ButtonComponent.vue';
-
+    import { openNewWindow } from '../utilis/openNewWindow.js';
```
Comment (PixelSculptor) [inline src/components/CertsComponent.vue]: I thought now that name for example 'openNewCart' would be more readable ;)

File: src/utilis/openNewWindow.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const openNewWindow = (link, params = '_blank') => {
```
Comment (PixelSculptor) [inline src/utilis/openNewWindow.js]: there is wrong letter: 'utils' not 'utilis'

**Review by PixelSculptor — APPROVED** [review]
Please rename this suggestions and then dont send me back, just merge

File: src/components/CertsComponent.vue (hunk 1 above)
Comment (glanttz) [reply src/components/CertsComponent.vue:22]: place in DB

Comment (glanttz) [reply src/components/CertsComponent.vue:22]: image
#

**Pull request merged**